docs/splunk-apis.md

```diff
@@ -0,0 +1,32 @@
+# Splunk APIs
+
+### 1. Retrieve Ingested Data:
+Ingest data in JSON format. Supports reading json data from file.
+
+```
+GET /api/search
+POST /api/search
+```
+
+Parameters:
+- `searchText`: The actual search query using Splunk syntax.
+- `indexName:` Specifies the index (like "traces") to search data from.
+
+Example:
+
+Using raw json:
+```bash
+# Send data in the request body to perform a search using Splunk QL.
+curl -X POST http://localhost:5122/api/search \
+-H "Content-Type: application/json" \
+-d '{
+  "startEpoch": "now-1h",
+  "endEpoch": "now",
+  "searchText": "*",
+  "indexName": "traces",
+  "queryLanguage": "Splunk QL"
+}'
+
```

Comment:
add a sample response. 
run a query locally and get the sample response and post here